close bazel ci/cd

i don't known how to fix bazel error:
`
ERROR: /home/runner/work/jobs/jobs/BUILD.bazel:5:13: Building libjava-maven-lib-class.jar (49 source files) failed: (Exit 1): java failed: error executing Javac command (from target //:java-maven-lib) external/rules_java~7.1.0~toolchains~remotejdk21_linux/bin/java '--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED' '--add-exports=jdk.compiler/com.sun.tools.javac.main=ALL-UNNAMED' ... (remaining 19 arguments skipped)
Note: Some messages have been simplified; recompile with -Xdiags:verbose to get full output
src/main/java/org/daming/jobs/base/shiro/JWTFilter.java:16: error: cannot access Filter
public class JWTFilter extends BasicHttpAuthenticationFilter {
                               ^
  class file for javax.servlet.Filter not found
src/main/java/org/daming/jobs/base/shiro/JWTFilter.java:16: error: cannot access ServletException
public class JWTFilter extends BasicHttpAuthenticationFilter {
       ^
  class file for javax.servlet.ServletException not found
src/main/java/org/daming/jobs/config/ShiroConfig.java:49: error: incompatible types: JWTFilter cannot be converted to Filter
        filterMap.put("jwt", new JWTFilter());
                             ^
src/main/java/org/daming/jobs/config/ShiroConfig.java:50: error: incompatible types: Map<String,jakarta.servlet.Filter> cannot be converted to Map<String,javax.servlet.Filter>
        factoryBean.setFilters(filterMap);
                               ^
Target //:jobs failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 40.059s, Critical Path: 8.44s
INFO: 108 processes: 10 internal, 95 linux-sandbox, 3 worker.
ERROR: Build did NOT complete successfully
`

i know why but i can not fix it. 
but i don't plan giving up bazel. i just remote the bazel ci/cd

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -51,28 +51,28 @@ jobs:
         run: cd ./src/main/react && yarn && yarn build
       - name: Build with Gradle
         run: ./gradlew build -x test
-  build-bazel:
-    needs: build
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        java-version: [ 21 ]
-        node-version: [ 20 ]
-        os: [ ubuntu-latest ]
-    steps:
-      - uses: actions/checkout@v3
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
-        with:
-          node-version: ${{ matrix.node-version }}
-      - name: Set up JDK
-        uses: actions/setup-java@v3
-        with:
-          distribution: 'zulu'
-          java-version: ${{ matrix.java-version }}
-      - name: Install yarn
-        run: npm install -g yarn
-      - name: Build with vite
-        run: cd ./src/main/react && yarn && yarn build
-      - name: Build with bazel
-        run: bazel build //:jobs
+  # build-bazel:
+  #   needs: build
+  #   runs-on: ${{ matrix.os }}
+  #   strategy:
+  #     matrix:
+  #       java-version: [ 21 ]
+  #       node-version: [ 20 ]
+  #       os: [ ubuntu-latest ]
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #     - name: Use Node.js ${{ matrix.node-version }}
+  #       uses: actions/setup-node@v1
+  #       with:
+  #         node-version: ${{ matrix.node-version }}
+  #     - name: Set up JDK
+  #       uses: actions/setup-java@v3
+  #       with:
+  #         distribution: 'zulu'
+  #         java-version: ${{ matrix.java-version }}
+  #     - name: Install yarn
+  #       run: npm install -g yarn
+  #     - name: Build with vite
+  #       run: cd ./src/main/react && yarn && yarn build
+  #     - name: Build with bazel
+  #       run: bazel build //:jobs

